### PR TITLE
update: Request interface of Fetch API

### PR DESCRIPTION
**Pull request by fwqaaq**

### CODE REVIEW

File: files/zh-cn/web/api/request/index.md [hunk 1]
```diff
@@ -3,117 +3,119 @@ title: Request
...
 ---
 
-{{APIRef("Fetch")}}{{SeeCompatTable}}[Fetch API](/zh-CN/docs/Web/API/Fetch_API) 的 **`Request`**接口？用来表示资源请求。
+{{APIRef("Fetch API")}}
 
-你可以使用 {{domxref("Request.Request()")}} ?构造函数创建一个 `Request` 对象，但是你可能会遇到一个 Request 对象作为其他 API 的操作被返回，比如一个 `service worker` 的{{domxref("FetchEvent.request")}}。
+[Fetch API](/zh-CN/docs/Web/API/Fetch_API) 的 **`Request`** 接口用来表示资源请求。
 
-## 构造器
+你可以使用 {{domxref("Request.Request","Request()")}} 构造函数创建一个新的 `Request` 对象，但是你可能会遇到作为其他 API 操作结果返回的 Request 对象，比如一个 `service worker` 的 {{domxref("FetchEvent.request")}}。
```
Comment (fwqaaq) [inline files/zh-cn/web/api/request/index.md:10]:
```suggestion
你可以使用 {{domxref("Request.Request","Request()")}} 构造函数创建一个新的 `Request` 对象，但是你可能会遇到作为其他 API 操作结果返回的 Request 对象，比如 `service worker` 的 {{domxref("FetchEvent.request")}}。
```

File: files/zh-cn/web/api/request/index.md [hunk 2]
```diff
@@ -126,8 +128,8 @@ fetch(myRequest)
 
 {{Compat}}
 
-## 相关链接
+## 参见
 
 - [ServiceWorker API](/zh-CN/docs/Web/API/ServiceWorker_API)
-- [HTTP access control (CORS)](/zh-CN/docs/Web/HTTP/Access_control_CORS)
+- [跨源资源共享（CORS）](/zh-CN/docs/Web/HTTP/Access_control_CORS)
```
Comment (jasonren0403) [inline files/zh-cn/web/api/request/index.md:134]:
```suggestion
- [跨源资源共享（CORS）](/zh-CN/docs/Web/HTTP/CORS)
```

**Review by jasonren0403 — CHANGES_REQUESTED** [review]
some suggestions

File: files/zh-cn/web/api/request/index.md [hunk 3]
```diff
@@ -3,117 +3,119 @@ title: Request
...
+- {{domxref("Request.referrer")}} {{ReadOnlyInline}}
+  - : 包含请求的 referrer（例如，`client`）。
+- {{domxref("Request.referrerPolicy")}} {{ReadOnlyInline}}
+  - : 包含请求的 referrer 策略（例如，`no-referrer`）。
+- {{domxref("Request.signal")}} {{ReadOnlyInline}}
+  - : 返回与请求相关联的 {{domxref("AbortSignal")}}。
+- {{domxref("Request.url")}} {{ReadOnlyInline}}
+  - : 包含请求的 URL。
+
+## 实例方法
```
[the remark this answers is not in the document]
Comment (fwqaaq) [reply files/zh-cn/web/api/request/index.md:48]: ??

File: files/zh-cn/web/api/request/index.md [hunk 4]
```diff
@@ -3,117 +3,119 @@ title: Request
...
+  - : 返回 promise，使用请求主体的 {{domxref("FormData")}} 表示来兑现。
 - {{domxref("Body.json()")}}
-  - : 返回解决一个{{domxref("JSON")}}表示的请求主体的 promise.
+  - : 返回 promise，使用解析请求体的结果作为 {{JSxRef("JSON")}} 来兑现。
 - {{domxref("Body.text()")}}
-  - : 返回解决一个{{domxref("USVString")}}(文本) 表示的请求主体的 promise.
+  - : 返回 promise，使用请求主体的文本表示来兑现。
 
-注意：这些 Body 功能只能运行一次; 随后的调用将通过空 strings/ ArrayBuffers 解析。
+>**备注：** 这些 Body 上的函数只能运行一次; 随后的调用将通过空 strings/ArrayBuffers 解析。
```
[the remark this answers is not in the document]
Comment (fwqaaq) [reply files/zh-cn/web/api/request/index.md:63]:
```suggestion
>**备注：** 这些请求主体上的函数只能运行一次，随后的调用将兑现为空字符串或空的 ArrayBuffer。
```

File: files/zh-cn/web/api/request/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jasonren0403) [reply files/zh-cn/web/api/request/index.md:48]: image

File: files/zh-cn/web/api/request/index.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (jasonren0403) [reply files/zh-cn/web/api/request/index.md]: 哦对，这里有点问题 🤔 我有时间去提个 PR

image

File: files/zh-cn/web/api/request/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (fwqaaq) [reply files/zh-cn/web/api/request/index.md:48]: 👌

**Comment by jasonren0403** [comment]
temporaily hold until https://github.com/mdn/content/pull/27697 resolves

File: files/zh-cn/web/api/request/index.md [hunk 5]
```diff
@@ -3,117 +3,119 @@ title: Request
...
+  - : 创建一个当前 `Request` 对象的副本。
+- {{domxref("Request.formData()")}}
+  - : 返回 promise，其兑现值为请求主体的 {{domxref("FormData")}} 表示形式。
+- {{domxref("Request.json()")}}
+  - : 返回 promise，其兑现值为请求主体经过 {{JSxRef("JSON")}} 解析的结果。
+- {{domxref("Request.text()")}}
+  - : 返回 promise，其兑现值为请求主体的文本表示形式。
 
-注意：这些 Body 功能只能运行一次; 随后的调用将通过空 strings/ ArrayBuffers 解析。
+>**备注：** 这些请求主体上的函数只能运行一次，随后的调用将兑现为空字符串或空的 ArrayBuffer。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/request/index.md:63]:
```suggestion
> **备注：** 这些请求主体上的函数只能运行一次，随后的调用将出现 TypeError 错误，表明请求主体流已被使用。
```

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**